### PR TITLE
Dit 667 - Classification Officers can Cancel a BTI ruling (status change only) 

### PR DESCRIPTION
**Pull request by SteveSugden**

### CODE REVIEW

File: app/uk/gov/hmrc/tariffclassificationfrontend/audit/AuditService.scala [hunk 1]
```diff
@@ -47,7 +47,7 @@ class AuditService @Inject()(auditConnector: DefaultAuditConnector) {
   def auditCaseReOpen(oldCase: Case, updatedCase: Case, operator: Operator)
                      (implicit hc: HeaderCarrier): Unit = {
     sendExplicitAuditEvent(
-      auditEventType = CaseReopened,
+      auditEventType = CaseReopen,
```
Comment (gokyo) [inline app/uk/gov/hmrc/tariffclassificationfrontend/audit/AuditService.scala]: I suggest to have `ed` at the end, for consistency with other events

File: app/uk/gov/hmrc/tariffclassificationfrontend/audit/AuditService.scala [hunk 2]
```diff
@@ -108,11 +116,12 @@ class AuditService @Inject()(auditConnector: DefaultAuditConnector) {
 
 object AuditPayloadType {
 
-  val CaseReopened = "caseReopened"
+  val CaseReopen = "caseReopened"
```
Comment (gokyo) [inline app/uk/gov/hmrc/tariffclassificationfrontend/audit/AuditService.scala]: could you add `ed`?

**Comment by ed0906** [comment]
Looks good to me

File: app/uk/gov/hmrc/tariffclassificationfrontend/views/cancel_ruling.scala.html [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.partials.case_heading
+@import views.html.helper.{CSRF, form}
+
+@(c: Case)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@main(bodyClasses = None) {
+
+    @case_heading(c)
+
+    <div class="grid-row">
```
Comment (ed0906) [inline app/uk/gov/hmrc/tariffclassificationfrontend/views/cancel_ruling.scala.html]: I'd maybe add an ID to the top level container to help with acceptance tests

File: app/uk/gov/hmrc/tariffclassificationfrontend/audit/AuditService.scala (hunk 2 above)
Comment (ed0906) [reply app/uk/gov/hmrc/tariffclassificationfrontend/audit/AuditService.scala]: Add Me? :D

Comment (SteveSugden) [reply app/uk/gov/hmrc/tariffclassificationfrontend/audit/AuditService.scala]: Same..

File: app/uk/gov/hmrc/tariffclassificationfrontend/views/cancel_ruling.scala.html (hunk 3 above)
Comment (SteveSugden) [reply app/uk/gov/hmrc/tariffclassificationfrontend/views/cancel_ruling.scala.html]: done

**Pull request merged**